src/domains/load-test/02_iam.tf

```diff
@@ -0,0 +1,9 @@
+data "azuread_group" "adgroup_developers" {
+  display_name = "${local.product}-adgroup-developers"
+}
+
+resource "azurerm_role_assignment" "adgroup_devevelopers_to_grafana_admin" {
+  scope                = data.azurerm_subscription.current.id
+  role_definition_name = "Grafana Admin"
```

Comment:
[ later we create groups with different permissions too](https://learn.microsoft.com/en-us/azure/managed-grafana/how-to-share-grafana-workspace#assign-an-admin-viewer-or-editor-role-to-a-user) right ?